RecFCCeeCalorimeter/tests/options/run_thetamodulemerged.py

```diff
@@ -0,0 +1,779 @@
+                                           readoutNames=[ecalBarrelReadoutName],
+                                           layerFieldNames=["layer"],
+                                           thetaRecalcWeights=[
+                                               # [-1, 3.0, 3.0, 3.0, 4.25, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0]  # -1 : use linear weights
+                                               [-1, 3.0, 3.0, 3.0, 4.25, 4.0, 4.0, 4.0, 4.0, 4.0, 4.0]  # -1 : use linear weights
+                                           ],
+                                           OutputLevel=INFO
+                                           )
+
+calibrateCaloClusters = CalibrateCaloClusters("calibrateCaloClusters",
+                                              # inClusters=createClusters.clusters.Path,
+                                              inClusters="Augmented" + createClusters.clusters.Path,
+                                              outClusters="Calibrated" + createClusters.clusters.Path,
+                                              systemIDs=[4],
+                                              numLayers=[11],
+                                              firstLayerIDs=[0],
+                                              readoutNames=[
+                                                  ecalBarrelReadoutName],
+                                              layerFieldNames=["layer"],
+                                              calibrationFile=os.environ['FCCBASEDIR'] + "/LAr_scripts/data/lgbm_calibration-CaloClusters-v03.onnx",
+                                              OutputLevel=INFO
+                                              )
+
+# TOPO CLUSTERS PRODUCTION
+createTopoInput = CaloTopoClusterInputTool("CreateTopoInput",
+                                           ecalBarrelReadoutName=ecalBarrelReadoutName,
+                                           ecalEndcapReadoutName="",
+                                           ecalFwdReadoutName="",
+                                           hcalBarrelReadoutName=hcalBarrelReadoutName2,
+                                           hcalExtBarrelReadoutName="",
+                                           hcalEndcapReadoutName="",
+                                           hcalFwdReadoutName="",
+                                           OutputLevel=INFO)
+
+createTopoInput.ecalBarrelCells.Path = ecalBarrelPositionedCellsName
+createTopoInput.ecalEndcapCells.Path = "emptyCaloCells"
+createTopoInput.ecalFwdCells.Path = "emptyCaloCells"
+createTopoInput.hcalBarrelCells.Path = hcalBarrelPositionedCellsName2
+createTopoInput.hcalExtBarrelCells.Path = "emptyCaloCells"
+createTopoInput.hcalEndcapCells.Path = "emptyCaloCells"
+createTopoInput.hcalFwdCells.Path = "emptyCaloCells"
+cellPositionHcalBarrelNoSegTool = None
+cellPositionHcalExtBarrelTool = None
+
+neighboursMap = "/LAr_scripts/data/neighbours_map_v03_ecalB_thetamodulemerged.root"
```

Comment:
This will likely not work (the root file will not be found), or am I missing something? Can you maybe streamline this config file to create simple clusters (e.g. with sliding window without the need for these rootfiles) so that the test run?

Reply:
this actually works because in the following lines the topoclustering algorithms are not scheduled, see eg https://github.com/HEP-FCC/k4RecCalorimeter/pull/72/files#diff-262cb6467fba07c8f5498412934192dd3a79eec0225820307b2b8464a1538b07R748

We just took our full script and commented away a few lines to make the test run, rather than spending more time and drop everything that is not needed ;)